lib/hanami/assets/helpers.rb

```diff
@@ -125,9 +131,11 @@ def self.included(base)
       #   <%= javascript 'application' %>
       #
       #   # <script src="https://assets.bookshelf.org/assets/application-28a6b886de2372ee3922fcaf3f78f2d8.js" type="text/javascript"></script>
-      def javascript(*sources)
+      def javascript(*sources, **options)
```

Comment:
I don’t think it should be `true` by default, since the `async` option needs to be set, i.e. it is a positive decision.

The option needs to be implemented here otherwise it won’t be available to the helper. The underlying `#script` in the HTML helper could be extended as well, but that is obviously outside the scope of this patch (being that it is in a different repository). Ultimately, the option should be available to both, and I can add it to `hanami-helpers` too.
  
> On 14 Feb 2016, at 22:01, Trung Lê <notifications@github.com> wrote:
> 
> In lib/hanami/assets/helpers.rb <https://github.com/hanami/assets/pull/46#discussion_r52849491>:
> 
> > @@ -125,9 +131,11 @@ def self.included(base)
> >        #   <%= javascript 'application' %>
> >        #
> >        #   # <script src="https://assets.bookshelf.org/assets/application-28a6b886de2372ee3922fcaf3f78f2d8.js" type="text/javascript"></script>
> > -      def javascript(*sources)
> > +      def javascript(*sources, **options)
> What do you think about a default hash async: true instead?
> 
> And why do you implement the option here at this high level than in the lower level of script?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/hanami/assets/pull/46/files#r52849491>.
>